modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -52,6 +52,8 @@
     protected String groupId = "io.swagger";
     protected String artifactId = "swagger-java";
     protected String artifactVersion = "1.0.0";
+    protected String licenseName = "Apache License, Version 2.0";
```

Comment:
I'll change this to `Unlicense` instead based on the discussion in https://github.com/swagger-api/swagger-codegen/issues/2963